### ISSUE TITLE
Set Branching Strategy

### ISSUE THREAD

**Issue by dibley1973** [dibley1973/the-train-exchange#2]
**As a** Project manager,
**I want** to set the branching strategy and add it to the wiki,
**So that** new features and bugs can be merged into the trunk in a controlled manner

Suggest the following branching strategy, or similar.

- Release from **main**.
- Release candidate from **develop**
- Use **bugfix** branches off of **develop** for fixing bugs, merge back to **develop** when PR'd and fixed
- Use **feature** branches off of **develop** for small features, merge back to **develop** when PR'd and complete
- Use **collab** branches off of **develop** for larger features with small feature branches off the collab. do not merge until the whole large feature is complete
- Use **hotfix** off **main** for emergency bug fixes, merge back to **main** when PR'd and fixed

```
main
  |
  |-- develop
  |       |
  |       |-- feature 1
  |       |
  |       |-- bugfix 1
  |       |
  |       |-- colab
  |       |       |
  |       |       | -- feature 2
  |       |       |
  |       |       | -- feature 3
  |       |
  |       |  -- feature 4
  | 
  | -- hotfix
```
